ext/libyuv.cmd

```diff
@@ -22,6 +22,6 @@ git checkout 464c51a0
 mkdir build
 cd build
 
-cmake -G Ninja -DCMAKE_BUILD_TYPE=Release ..
+cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DBUILD_SHARED_LIBS=OFF ..
```

Comment:
Yannis: I believe `-DBUILD_SHARED_LIBS=OFF` should not be added, because libyuv's CMakeLists.txt controls the kind of library to build by the target name (yuv vs. yuv_shared). Please check this change.

Reply:
Oh interesting. And we've only ever linked against yuv. I'll remove the flag since it doesn't do anything, but keep `- DCMAKE_POSITION_INDEPENDENT_CODE=ON`?

Reply:
`-DCMAKE_POSITION_INDEPENDENT_CODE=ON` as the library does not handle it